recipes/dooit/recipe.yaml

```diff
@@ -0,0 +1,61 @@
+context:
+  name: dooit
+  version: "3.1.0"
+  python_min: "3.9"
+
+package:
+  name: ${{ name|lower }}
+  version: ${{ version }}
+
+source:
+  url: https://pypi.org/packages/source/${{ name[0] }}/${{ name }}/${{ name }}-${{ version }}.tar.gz
+  sha256: ceb0faa4d5a93976a3895505b66c11a80a16c616d6a7d20bbfa9e6c2dbe7c05a
+
+build:
+  noarch: python
+  script: python -m pip install . -vv
+  number: 0
+  python:
+    entry_points:
+      - dooit = dooit.__main__:main
+
+requirements:
+  host:
+    - python ${{ python_min }}
+    - pip
+    - poetry-core
+  run:
+    - python >= ${{ python_min }}
```

Comment:
No functional difference (I think), but remove the space: `- python >=${{ python_min }}`